.bazelrc

```diff
@@ -7,7 +7,7 @@ common --check_direct_dependencies=off
 
 # bazel 7 is somewhat forgiving for glob patterns that don't
 # match, but bazel 8 will be strict. So start now.
-common --incompatible_disallow_empty_glob
+#common --incompatible_disallow_empty_glob
```

Comment:
... don't do this.

Reply:
didn't want to, but it was getting in the way of my immediate objectives. I figured you might have a solution to this...